src/schema/v2/viewingRoom.ts

```diff
@@ -68,11 +69,7 @@ export const ViewingRoomType = new GraphQLObjectType<any, ResolverContext>({
     const { PartnerArtworks } = require("schema/v2/partner/partnerArtworks")
 
     return {
-      internalID: {
-        description: "A type-specific ID likely used as a database ID.",
-        type: new GraphQLNonNull(GraphQLID),
-        resolve: ({ id }) => id,
-      },
+      ...InternalIDField,
```

Comment:
**suggestion / non-blocking**: Since this is just a single field and we're not spreading multiple fields, pulling in the specific field definition might be fine too:

```ts
internalID: InternalIDField
```